mautic/mautic
# [UX] Search contact name in Google

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

this pr adds a new option to the contact actions dropdown that allows users to search for a contact's name on google directly from the mautic interface. this feature aims to enhance the crm capabilities of mautic by providing a quick and convenient way for users to gather additional information about their contacts through a simple web search. 

the implementation is straightforward and should have minimal impact on the existing codebase. this improvement will benefit mautic users who frequently need to research their contacts and can help streamline their workflow.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > Create new > Save & Close > Click on the dropdown to see more actions
3. Click on the Search in Google option to open a new search page
